app/controllers/assignments_controller.rb

```diff
@@ -77,6 +77,10 @@ def student_interface
 
     @student = current_user
     @grouping = @student.accepted_grouping_for(@assignment.id)
+    @penalty = SubmissionRule.find_by_assignment_id(@assignment.id)
+    
+    # sort if it's penaltydecay or penalty period
+    @enum_penalty = Period.find_all_by_submission_rule_id(@penalty.id).sort_by{|obj| obj.hours}
```

Comment:
Line is too long. [95/80]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.